packages/block-editor/src/components/border-radius-control/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	RangeControl,
+	__experimentalParseUnit as parseUnit,
+	__experimentalUseCustomUnits as useCustomUnits,
+} from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AllInputControl from './all-input-control';
+import InputControls from './input-controls';
+import LinkedButton from './linked-button';
+import {
+	getAllValue,
+	getAllUnit,
+	hasDefinedValues,
+	hasMixedValues,
+} from './utils';
+
+const DEFAULT_VALUES = {
+	topLeft: null,
+	topRight: null,
+	bottomLeft: null,
+	bottomRight: null,
```

Comment:
Any reason for the `null` value here? Is it the same as assigning `0`? If so, could we keep the same type for consistency?

Reply:
Good question. This is deliberately `null`. This allows the split border radius controls to remain empty as `0` is otherwise a valid value. It also means we can omit unnecessary longhand CSS styles and inherit values from the theme.json or global styles.